### PR TITLE
Add Amber

### PR DESCRIPTION
**Pull request by faustinoaq**
adding :

- [Amber](https://amberframework.org/) as Web Framework & Web Server

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -78,6 +78,17 @@
 			"icon": "ebis.png",
 			"website": "http://www.ebis.ne.jp"
 		},
+		"Amber": {
+			"cats": [
+				"18",
+				"22"
+			],
+			"headers": {
+				"X-Powered-By": "Amber"
```
Comment (jvoisin) [inline src/apps.json]: Please use `"X-Powered-By": "^Amber$"` instead, to speed up a bit the matching.

File: src/apps.json [hunk 2]
```diff
@@ -78,6 +78,17 @@
...
+			},
+			"icon": "amber.png",
+			"website": "http://amberframework.org"
```
Comment (jvoisin) [inline src/apps.json]: Please use `https` instead ♥

Comment (faustinoaq) [reply src/apps.json]: Done! 👍

File: src/apps.json (hunk 1 above)
Comment (faustinoaq) [reply src/apps.json]: Done! 👍

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**